test/e2e/push_test.go

```diff
@@ -163,6 +165,7 @@ var _ = Describe("Podman push", func() {
 	})
 
 	It("podman push to docker-archive", func() {
+		SkipIfRemote("Rootless push does not support docker-archive transport")
```

Comment:
Should this be SkipIfRootless? Or should it say Remote push?

Reply:
Yes this should say Remote push does not support.  Will wait to see what tests pass.